### PR TITLE
Feature/ceiroa/fixing tests adv

### PR DESCRIPTION
**Pull request by ceiroa**
Disabling 4 bucket-related tests if ADV is installed. ADV doesn't support Bucket model.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/classes/RLLP_OppRollup_TEST.cls [hunk 1]
```diff
@@ -36,12 +36,6 @@
 */
 @isTest
 public with sharing class RLLP_OppRollup_TEST {
-
-    /*********************************************************************************************************
```
Comment (weizsforg) [inline src/classes/RLLP_OppRollup_TEST.cls]: Hi @ceiroa Can you please give me some context about why we need to delete this line?

File: src/classes/RLLP_OppRollup_TEST.cls [hunk 2]
```diff
@@ -131,7 +125,6 @@ public with sharing class RLLP_OppRollup_TEST {
         //skip the test if Advancement is installed
         if(ADV_PackageInfo_SVC.useAdv()) return;
 
-    	if (strTestOnly != '*' && strTestOnly != 'testGivingRollupOne2One') return;
```
Comment (weizsforg) [inline src/classes/RLLP_OppRollup_TEST.cls]: Hi @ceiroa Similar to above. Can you please give me some context about why we need to delete this line? Also, I saw we delete the same line in other methods in this class as well.

**Comment by weizsforg** [comment]
Hi @ceiroa , I saw some code in RLLP_OppRollup_TEST class got deleted. Is that on purpose?

File: src/classes/RLLP_OppRollup_TEST.cls (hunk 2 above)
Comment (ceiroa) [reply src/classes/RLLP_OppRollup_TEST.cls]: We don't have to, but these lines in each test method were used just to allow us to run only one method during development by modifying the strTestOnly variable. That was before Salesforce allowed you to choose which methods in a test class to run. They are not needed any more, so I thought I'd clean them up.

**Comment by weizsforg** [comment]
Hi @ceiroa , I assigned this PR to @annejackson. Is there anything to test? or we can ask @annejackson to just merge it?

**Comment by ceiroa** [comment]
Nothing to test. Let’s go ahead and merge. Thanks!

-Carlos

On Jul 20, 2018, at 5:32 PM, weizsf <notifications@github.com> wrote:

Hi @ceiroa <https://github.com/ceiroa> , I assigned this PR to @annejackson
<https://github.com/annejackson>. Is there anything to test? or we can ask
@annejackson <https://github.com/annejackson> to just merge it?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/SalesforceFoundation/Cumulus/pull/3507#issuecomment-406743019>,
or mute the thread

**Comment by annejackson** [comment]
Merged.

**Pull request merged**